Dashboard/app/js/lib/components/devices/AssignmentsEditView/__partials/SidebarDropdown.jsx

```diff
@@ -40,14 +40,16 @@ export default class SidebarDropdow extends React.Component {
 
         <div style={panelStyle} className="sidebar-panel">
           {devices.map(device => (
-            <button type="button">{device.name}</button>
+            <button key={device.id} type="button">
```

Comment:
One more thought here is that the list of devices is probably links rather than buttons no?

Reply:
Well yes. But links need `href` links. And we don't have them

Reply:
Maybe we can implement for now without the href but with associated actions. So whenever we replace the router we fill in the links?

Reply:
That works, but `eslint` will complain about it

Reply:
lets disable the complaint for now with a note to reenable it when links are in place.